client/shipping-label/state/actions.js

```diff
@@ -1,15 +1,99 @@
-import * as FormActions from 'settings/state/actions';
+import saveForm from 'lib/save-form';
+import noop from 'lodash/noop';
+import fill from 'lodash/fill';
+import flatten from 'lodash/flatten';
+import omit from 'lodash/omit';
+import moment from 'moment';
+import printDocument from 'lib/utils/print-document';
+import * as NoticeActions from 'state/notices/actions';
 
 export const OPEN_PRINTING_FLOW = 'OPEN_PRINTING_FLOW';
 export const EXIT_PRINTING_FLOW = 'EXIT_PRINTING_FLOW';
+export const UPDATE_ADDRESS_VALUE = 'UPDATE_ADDRESS_VALUE';
+export const UPDATE_PACKAGE_WEIGHT = 'UPDATE_PACKAGE_WEIGHT';
+export const UPDATE_RATE = 'UPDATE_RATE';
+export const UPDATE_PAPER_SIZE = 'UPDATE_PAPER_SIZE';
+export const PURCHASE_LABEL_REQUEST = 'PURCHASE_LABEL_REQUEST';
+export const PURCHASE_LABEL_RESPONSE = 'PURCHASE_LABEL_RESPONSE';
 
-export const openPrintingFlow = () => ( dispatch ) => {
-	dispatch( { type: OPEN_PRINTING_FLOW } );
-	dispatch( FormActions.setFormProperty( 'pristine', false ) );
-	dispatch( FormActions.nextStep() );
+export const openPrintingFlow = () => {
+	return { type: OPEN_PRINTING_FLOW };
 };
 
-export const exitPrintingFlow = () => ( dispatch ) => {
-	dispatch( { type: EXIT_PRINTING_FLOW } );
-	dispatch( FormActions.resetFlow() );
+export const exitPrintingFlow = () => {
+	return { type: EXIT_PRINTING_FLOW };
+};
+
+export const updateAddressValue = ( group, name, value ) => {
```

Comment:
`group` would be either `origin` or `destination`, so to change the origin postcode this action would be called like `updateAddressValue( 'origin', 'postcode', '12345' )`